docs/usage/from-a-terminal.rst

```diff
@@ -288,23 +289,29 @@ Don't want to perform some netinfo lookup ? This argument is for you.
 Activates or disables the usage of our SPECIAL and extra
 rules whether possible.
 
-**Default value:** :code:`True`
+**Default value:** :code:`special: True`
 
 Don't want to use/apply the SPECIAL rules - which are explained in the source
 column section? This argument disables them all.
 
+To read more about which `Special Rules <../responses/source.html#special>`_
+is applied.
 
 :code:`--whois-lookup`
 """"""""""""""""""""""
+.. versionadded:: 4.0.0
 
 Activates or disables the usage of the WHOIS record
 (or better said the expiration date in it) whether possible.
 
 **Default value:** :code:`True`
 
-Don't want to use or take into consideration the results from :code:`whois`?
+Don't want to use or take the :code:`whois` date into consideration?
```

Comment:
let's be more precise: `the WHOIS expiration date [...]`

Reply:
The hole section, needs a rewrite..... Currently I'm not working on the text as much as the order, getting it up to date with version 4... only severe cases that court my eyes get a first touch of rewrites, but will take it into consideration